src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CreateCaseCallbackTest.java

```diff
@@ -40,12 +37,16 @@
     @Test
     void should_create_case_if_classification_new_application_with_documents_and_ocr_data() {
         setUpTransformation(getTransformationResponseBody("ok-no-warnings.json"));
+        setUpCcdCreateCase(
+            getCcdResponseBody("start-event.json"),
+            getCcdResponseBody("sample-case.json")
+        );
 
         postWithBody(getRequestBody("valid-new-application-with-ocr.json"))
             .statusCode(OK.value())
             .body("errors", empty())
             .body("warnings", empty())
-            .body("data.caseReference", matchesPattern(CASE_ID_REGEX));
+            .body("data.caseReference", equalTo("1539007368674134")); // from sample-case.json
     }
 
     @Test
```

Comment:
I've just noticed that this test expects the body to contain warnings in `errors` field.

Reply:
Let me return a warning then. Makes sense. It all comes down to this "proper way of handling response from service" matter